<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeDialog</class>
 <widget class="QDialog" name="MergeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Merge</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="msg">
     <property name="text">
      <string>Please select the branches to merge, and the conflict resolution method:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="srcLabel">
     <property name="text">
      <string>Source branch</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="Tao::BranchSelectionComboBox" name="srcBranch"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="atLabel">
     <property name="text">
      <string>at</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="Tao::CommitSelectionComboBox" name="srcCommit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="conflictLabel">
     <property name="text">
      <string>Conflict resolution</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QRadioButton" name="manual">
     <property name="text">
      <string>Manual</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QRadioButton" name="ours">
     <property name="text">
      <string>Ours</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QRadioButton" name="theirs">
     <property name="text">
      <string>Theirs</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="Tao::BranchSelectionComboBox" name="dstBranch"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dstLabel">
     <property name="text">
      <string>Destination branch</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tao::BranchSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header>branch_selection_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>Tao::CommitSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header>commit_selection_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MergeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MergeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
